cypress.config.js

```diff
@@ -0,0 +1,20 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    fixturesFolder: false,
+    env: {
+      API_URL: 'http://localhost:3001/',
+    },
+  },
+  component: {
+    devServer: {
+      framework: 'create-react-app',
+      bundler: 'webpack',
+    },
+    indexHtmlFile: 'frontend/cypress/support/component-index.html',
+    specPattern: 'frontend/src/**/*.cy.{js,jsx,ts,tsx}',
+    supportFile: 'frontend/cypress/support/component.js'
+  },
+})
```

Comment:
É uma boa prática deixar uma linha em branco ao final de cada arquivo, para que o código seja melhor renderizado em serviços como o GitHub, onde quando não há tal linha em branco, um ícone é exibido ao final de cada arquivo, informando _No newline at end of file_.

```suggestion
})

```